tests/run.sh

```diff
@@ -12,5 +12,15 @@ function finish {
 }
 trap finish EXIT
 
-sleep 5
```

Comment:
This sleep was added to allow the coverage instrumentation to complete in server.js above. Without it, race issues __may__ occur.

Reply:
Ok, I had removed it as we have much longer timeouts and wait behaviour for response to `/_init` from the server. Will re-add.